src/hotspot/cpu/arm/c1_LIRGenerator_arm.cpp

```diff
@@ -363,7 +363,7 @@ void LIRGenerator::set_card(LIR_Opr value, LIR_Address* card_addr) {
   }
 }
 
-void LIRGenerator::CardTableBarrierSet_post_barrier_helper(LIR_OprDesc* addr, LIR_Const* card_table_base) {
+void LIRGenerator::CardTableBarrierSet_post_barrier_helper(LIR_Opr addr, LIR_Const* card_table_base) {
   assert(addr->is_register(), "must be a register at this point");
```

Comment:
The above rename seems unnecessary.

Reply:
It is actually necessary. Due to `typedef LIR_Opr LIR_OprDesc;`, `LIR_OprDesc*` is equivalent to `LIR_Opr*`, which breaks build.